bin/release

```diff
@@ -4,17 +4,12 @@
 #
 # Assumes bin/prep-release was run and the PR merged.
 #
-# Usage: bin/release VERSION
+# Usage: bin/release
 #
 ###
 set -e
 
-if [ -z "$1" ]; then
-  echo "usage: bin/release VERSION" >&2
-  exit 64
-fi
-
-version=$1
+version=$(< VERSION)
 
 git checkout master
```

Comment:
The reason it worked this way, is that this script doesn't assume you've pulled master since the PR was merged -- that's done here.

Not saying what you're picturing won't or can't work, but it might need a few more changes to ensure running this script doesn't release the existing version.